bonito/models/configs/dna_r9.4.1@v3.1.toml

```diff
@@ -22,5 +22,27 @@ rnn_type = "lstm"
 activation = "swish"
 blank_score = 2.0
 
+attn_layers = [ 3, 4 ]    # the list of layers (1 - 5) after which to play an attention block, right after the LSTM layer
+attn_dropout = 0.1        # attention dropout, keep at 0.1
+ff_dropout = 0.1          # feedforward dropout, keep at 0.1
+num_attn_heads = 1        # number of attention heads, which should be kept at 1 for single-head attention, but can be increased to > 1 to turn on multi-head attention
+dim_attn_head = 64        # dimension per attention head, should just keep at 64, but can be lowered to 32 for further efficiency / perf tradeoff
+
+use_isab_attn = false     # whether to use ISAB attention (induced-set attention block from the Set Transformers paper)
```

Comment:
if you were to set this to true, the number of attention heads need to be increased to 4 or above. a good starting config would be

```
num_attn_heads = 4
dim_attn_head = 64
use_isab_attn = true
isab_num_latents = 6
```